src/index.html

```diff
@@ -2,16 +2,190 @@
+          <input
+            type="number"
+            name="age"
+            id="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </div>
+
+        <div class="field__wrapper">
+          <label for="date">Full date of birth: </label>
+
+          <input
+            type="date"
+            name="date"
+            id="date"
+          >
+        </div>
+
+        <label for="check">I accept the term of the agreement </label>
+
+        <input
+          type="checkbox"
+          value="check"
+          name="check"
+          id="check"
+        >
+      </fieldset>
+
+      <fieldset class="box__style">
+        <legend>Registration</legend>
+
+        <div class="field__wrapper">
+          <label for="email">E-mail: </label>
+
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            multiple
+          >
+        </div>
+
+        <label for="user-password">Password: </label>
+
+        <input
+          name="user-password"
+          id="user-password"
+          type="password"
+          required autocomplete="off"
+        >
+      </fieldset>
+
+      <fieldset class="box__style">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="field__wrapper">
+          <label for="cat">Do you love cats? </label>
+
+          <input
+            type="radio"
+            id="yes"
+            name="cat"
+            value="Y"
+          >
+          <label for="yes">Yes</label>
+
+          <input
+            type="radio"
+            id="no"
+            name="cat"
+            value="N"
+          >
+          <label for="no">No</label>
+        </div>
+
+        <div class="field__wrapper">
+          <label for="color">What is your favorite color? </label>
+
+          <input
+            type="color"
+            name="color"
+          >
+        </div>
+
+        <div class="field__wrapper">
+          <label for="time">What time do you go to bed? </label>
+
+          <input
+            type="time"
+            name="time"
+            id="time"
+            step="2"
+          >
+        </div>
+
+        <div class="field__wrapper">
+          <label for="cars">What are your favorite brand of cars? </label>
+
+          <textarea
+            name="cars"
+            id="cars"
+            rows="4"
+            cols="2"
+          >BMW Audi Lada
+          </textarea>
+        </div>
+
+        <label for="range">How do you rate our work? </label>
+
+        <input
+          type="range"
+          name="range"
+          value="0"
+        >
+      </fieldset>
+
+      <fieldset class="box__style">
+        <legend>Additional info:</legend>
+
+        <div class="field__wrapper">
+          <label for="comment">Comments: </label>
+
+          <textarea
+            name="comment"
+            rows="2"
+            cols="20"
+          >
+          </textarea>
```

Comment:
**<textarea>** is a tricky tag to format:
It requires you to put a closing tag on the same line as the > of the opening tag, not to have default content, and be able to see the placeholder.
At the same time, if it has several attributes, you need to put each of them on its own line.
So, use the following formatting where > is moved 1 position left from the normal alignment:

```html
  <textarea
    class="..."
    placeholder="..."
 ></textarea>
```